docs/source/usage/workflows/probeParticles.rst

```diff
@@ -114,3 +114,4 @@ Known Limitations
 .. warning::
 
    If the probe particles are dumped in the file output, the instantaneous fields they recorded will be one time step behind the last field update (since our runOneStep pushed the particles first and then calls the field solver).
+   Adding the attributes probeE or probeB to a species will increase the particle memory footprint by ``3 * sizeof(float_X)`` byte per attribute and particle.
```

Comment:
``probeE`` or ``probeB`` (set text verbatim / code style)

Reply:
Maybe we should also clarify that the memory footprint is for this species. (Some new users may misunderstand as if it was for all species e.g. if all species had same attributes)